services/alerta/config.go

```diff
@@ -15,6 +15,8 @@ type Config struct {
 	InsecureSkipVerify bool `toml:"insecure-skip-verify" override:"insecure-skip-verify"`
 	// The authentication token for this notification, can be overridden per alert.
 	Token string `toml:"token" override:"token,redact"`
+	// The prefix for the Authentication field where the token is stored
+	TokenPrefix string `toml:"token-prefix" override:"token-prefix"`
```

Comment:
Can you add a comment here that if you are on an older version of Alerta you may want to set this to `Key`? Along those lines can you update the example config in `etc/kapacitor/kapacitor.conf` with this new option and comments as well.